<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <row/>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rand">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Рандом</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_max">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимум</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sz">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ср.значение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sz">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_puz">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка &quot;Пузырёк&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gnom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Гномья сортировка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fast">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Быстрая сортировка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_puz">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_gnom">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_fast">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>382</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка &quot;Расчестка&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_comb">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_monkey">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обезьянья сортировка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_monkey">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>460</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_duble">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удаление дубликатов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dubl">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_search">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите значение числа, которое вы хотите найти</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_res">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_res">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ячейка №</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во совпадений</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_count">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_const">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>116</width>
      <height>23</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>-999999</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_const">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить константой</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
